wsrep_api.h

```diff
@@ -246,6 +253,14 @@ extern int
 wsrep_gtid_print(const wsrep_gtid_t* gtid, char* str, size_t str_len);
 
 /*!
+ * Compare two GTIDs
+ * @return -1, 0, 1 if lhs is respectively smaller, equal, or greater than rhs
+ */
```

Comment:
Please add to description how is the comparison defined (gtid compared first etc.).